Mathlib/Algebra/Category/ModuleCat/Sheaf/Quasicoherent.lean

```diff
@@ -56,6 +58,94 @@ attribute [instance] Presentation.IsFinite.isFiniteType_generators
 
 end
 
+noncomputable section
+
+variable {C C' : Type (u')} [SmallCategory C] [SmallCategory C'] {J : GrothendieckTopology C}
+  {J' : GrothendieckTopology C'} {R : Sheaf J RingCat} {S : Sheaf J' RingCat}
+  [HasSheafify J AddCommGrpCat] [HasSheafify J' AddCommGrpCat]
+  [J.WEqualsLocallyBijective AddCommGrpCat] [J'.WEqualsLocallyBijective AddCommGrpCat]
+  [J.HasSheafCompose (forget₂ RingCat AddCommGrpCat)]
+  [J'.HasSheafCompose (forget₂ RingCat AddCommGrpCat)] {ι σ : Type u}
```

Comment:
I think the section would be clearer if variables were introduced in order of needing them, the first two definitions do not need `C'`, `J'`, `S` and their associated instances.